src/common/gamewrapper.ts

```diff
@@ -0,0 +1,59 @@
+import { Game } from 'boardgame.io';
+import { TurnOrder } from 'boardgame.io/core';
+
+function chooseRole({ G, ctx, playerID }: any, firstPlayer: string) { // TODO: type
+	G.firstPlayer = firstPlayer;
+}
+
+export function gameWrapper(game: any): Game<any> { // TODO: solve types
+	return {
+		setup: () => ({ ...game.setup(), firstPlayer: null, difficulty: null, winner: null }),
```

Comment:
As far as I see, difficulty is not yet implemented, only passed around / initialized. I would remove it from code until there is an actual usage.

Reply:
Good catch! Currently it is used (however it shouldn't), when someone starts the game. Created an issue for this one.